hash/thinkpad8xx_flop.xml

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<!DOCTYPE softwarelist SYSTEM "softwarelist.dtd">
+<!--
+license:CC0-1.0
+-->
+<softwarelist name="thinkpad8xx_flop" description="IBM Thinkpad 800 series software on floppy disks">
+
+	<!-- Operating systems -->
+
+	<software name="sol_251_ibm_ppc">
+		<description>Solaris MDB Boot 2.5.1 Release PowerPC Platform Edition (IBM Hardware Platform)</description>
+		<year>1996</year>
+		<publisher>Sun Microsystem</publisher>
+		<part name="flop" interface="floppy">
+			<feature name="part_number" value="702-4348-06"/>
+			<dataarea name="flop" size="168000">
+				<rom name="solaris_2_5_1_mdb_boot_ppc_ibm.img" size="168000" crc="b45832e2" sha1="be9e11364816ab5f3362396ed0f457d4a2e61188" />
+			</dataarea>
+		</part>
+	</software>
```

Comment:
This should be a part of the Solaris installation items, not a separate software list item.

Reply:
Fixed!